doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,923 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(air-serving-guide)=\n",
+    "\n",
+    "# Serve Ray AIR Predictor with `ModelWrapper`\n",
+    "\n",
+    "[Ray Serve](rayserve) is the recommended tool to deploy models trained with AIR.\n",
+    "\n",
+    "The core concept is the `ModelWrapper`. The `ModelWrapper` takes a [predictor](ray.air.predictor.Predictor) class and a [checkpoint](ray.air.checkpoint) and transforms them into a live HTTP endpoint. \n",
```

Comment:
Add a comment about why one should care about this doc.

"""
After training a model with Ray Train, you can serve a model using Ray Serve. In this guide, we will cover how to use Ray AIR's ModelWrapper, Predictor, and Checkpoint abstractions to quickly deploy a model for online inference.

In this guide, you will need to know a couple key concepts:
* A ``ModelWrapper`` (link to ref) is a X. It takes in a ``Predictor``... etc
* A ``Predictor`` (link to ref) is a XYZ. It is instantiated by taking in a Model Checkpoint. 
* A ``Checkpoint`` (link to ref) is an object that represents a serialized model.